regression/cbmc/Fixedbv7/test.desc

```diff
@@ -1,6 +1,6 @@
-CORE
+KNOWNBUG
```

Comment:
Please add a comment at the end of the file describing what is going wrong here. And likely also create an issue.